### DISCUSSION TITLE
Regarding generate restart files

### DISCUSSION THREAD

**Discussion by xuelingbo** [NCAR/hrldas#156] [Ideas]
Hi,
When compiling HRLDAS using gfortran, a restart file is generated at the initial time step of each month, regardless of which option I select for `RESTART_FREQUENCY_HOURS`. This problem persists when using the Fujitsu compiler on a supercomputer, and in addition, the simulation stops abruptly after the restart file is generated. For now, I have temporarily commented out the code related to the restart file to ensure that the simulation run is not interrupted. I think this may have something to do with the compiler.

**Comment by tslin2** [comment]
This may be a related issue. https://github.com/NCAR/hrldas/discussions/86
try to use a larger number hours so there are fewer restart file outputs, We will probably revise this part later.
Thanks
